spec/system/auth/sign_in_spec.rb

```diff
@@ -2,29 +2,29 @@
 
 require 'rails_helper'
 
-RSpec.feature 'Sign in', type: :feature do
+RSpec.describe 'Sign in' do
   let(:email) { 'zora@example.org' }
   let(:password) { '12345678' }
   let(:otp_enabled) { true }
   let!(:user) { create(:user, email: email, password: password, otp_enabled: otp_enabled) }
 
-  scenario 'editor tries to visit protected page' do
+  it 'editor tries to visit protected page' do
     visit dashboard_path
     expect(page).to have_current_path(sign_in_path)
   end
 
-  context 'without 2FA set up' do
+  describe 'without 2FA set up' do
     let(:otp_enabled) { false }
 
-    scenario 'editor signs in' do
+    it 'editor signs in' do
       visit sign_in_path
 
       # Enter incorrect credentials
       fill_in 'session[email]', with: 'zora@example.org'
       fill_in 'session[password]', with: 'abcdefgh'
       click_button 'Anmelden'
 
-      expect(page).to have_current_path(session_path)
+      expect(page).to have_current_path(sign_in_path)
```

Comment:
This was weird that this was passing... `session_path` is a:

```
session POST   /session(.:format)                                                                                sessions#create
```

I think we do _not_ expect the user to be on this path